src/Model/Command/ItemPropertySetup.php

```diff
@@ -58,6 +57,27 @@ public static function set(string $propertyName): self
         return new static($propertyName, self::PROPERTY_TYPE_SET);
     }
 
+    protected function setPropertyName(string $propertyName): void
+    {
+        Assertion::typeIdentifier($propertyName);
+
+        $this->propertyName = $propertyName;
+    }
+
+    protected function setPropertyType(string $propertyType): void
+    {
+        Assertion::choice($propertyType, [
```

Comment:
This is redundant, isn't it?

I cannot create invalid `ItemPropertySetup` cause private `constructor`.

And factory constructors are covered by tests.